app/assets/javascripts/questions.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```

Comment:
This is evidence you are using generators but not cleaning up after them. If you're not using coffeescript, remove the gem coffee-rails to prevent the generator generating coffeescript files. Or just remove them. There are also flags to the generators to skip assets and modules.